Content.Shared/Teleportation/Components/TeleportLocationsComponent.cs

```diff
@@ -55,10 +55,10 @@ public partial record struct TeleportPoint
 {
     [DataField]
     public string Location;
-    [DataField]
-    public NetEntity TelePoint;
+    [NonSerialized, DataField]
+    public EntityUid TelePoint;
```

Comment:
This isn't correct, because the parent type is also used in networking. You need to split these types.